string zoneName
string[] modelNames
std_msgs/Int8[] modelColors
std_msgs/Float32 distanceFromRobot
